Rust/mechanics_fast/Cargo.toml

```diff
@@ -11,3 +11,4 @@ crate-type = ["cdylib"]
 
 [dependencies]
 cgmath = "0.18.0"
+approx = "0.5.1"
```

Comment:
Do we need this? On my machine, the tests pass even if I remove it.

Reply:
Even if we don't need it, it can stay there, it's not going to hurt anyone by its presence.